src/hooks/use-suggestions.ts

```diff
@@ -644,7 +644,7 @@ const getOperatorByDataType = (dataType: string) => {
     })
 
     if (dataType === 'date' || dataType === 'datetime') {
-        const toExclude = ['$in', '$nin']
+        const toExclude = ['$eq', '$neq', '$in', '$nin']
```

Comment:
Some places it's $neq and some places it's $ne , can you align all to $ne ?

Reply:
done (I fugured parse-smart-query's use - $ne - should take precedence)